test/system/helpers.bash

```diff
@@ -521,5 +521,15 @@ function remove_same_dev_warning() {
     output=$(printf '%s\n' "${lines[@]}")
 }
 
+# run 'podman help', parse the output looking for 'Available Commands';
+# return that list.
+function podman_commands() {
```

Comment:
If you need to resubmit, could you prepend an underscore (`_podman_commands`) to mark this as a local internal helper function?